src/java/org/apache/cassandra/io/sstable/StorageHandler.java

```diff
@@ -64,6 +64,10 @@
         REPAIR,
         /** A request over forced by users to reload. */
         USER_REQUESTED,
+        /** SSTables might have been added or removed, regardless of a specific reason
+         * e.g. it could be compaction or flushing or regions being updated which caused
+         * new sstables to arrive */
+        SSTABLES_CHANGED
```

Comment:
this option was not added in CNDB-1707 (in which this enum was created). I don't know why it was omitted, I didn't find an explanation.

@jasonstack, was it only an oversight, or is there something to it?

Reply:
I think it's added after the remote storage port. https://github.com/riptano/bdp/pull/19943/files